peartree/summarizer.py

```diff
@@ -55,9 +59,13 @@ def generate_all_observed_edge_costs(trips_and_stop_times: pd.DataFrame
         tst_sub = trips_and_stop_times[tst_mask]
 
         # Just in case both directions are under the same trip id
-        for direction in [0, 1]:
-            dir_mask = (tst_sub.direction_id == direction)
-            tst_sub_dir = tst_sub[dir_mask]
+        for direction in [1]:
+            # Check if direction_id exists in source data
```

Comment:
Flesh out comment - "Support situations where`direction_id` is absent from the GTFS data. In such situations, include all trip and stop time data, instead of trying to split on that column (since it would not exist).